guides/common/modules/proc_associating-partition-tables-with-disk-encryption.adoc

```diff
@@ -0,0 +1,15 @@
+[id="associating-partition-tables-with-disk-encryption_{context}"]
+= Associating partition tables with disk encryption
+
+{Project} contains partition tables that encrypt the disk of your host using Clevis/Tang during host provisioning.
+You have to associate the partition table with your operating system entry and assign it to your host group or select it manually during provisioning.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Provisioning Setup* > *Operating Systems*.
+. Select your {client-os} entry.
+. On the *Partition Table* tab, associate the `{client-provisioning-template-type} default encrypted` to your operating system entry.
```

Comment:
Will the meaning of "associate... to your operating system entry" be clear to the user? I am having trouble visualizing it.

Reply:
I think so, yes. For every OS that you want to provision/manage, users have to associate various entities such as provisioning templates, architectures, installation media, etc.

I would also be OK to replace associate -> select; this IMHO we always use associate for this type of UI (random example from Foreman 3.10 RC):

![image](https://github.com/theforeman/foreman-documentation/assets/12595287/63f7c158-9b9d-4f41-bec2-68e8521fb6b0)

Reply:
Now I understand. Patternfly calls it a [dual list selector](https://www.patternfly.org/components/dual-list-selector). I will add a 'suggestion' just to show you how I would document this action. Feel free to ignore.

Reply:
```suggestion
. On the *Partition Table* tab, select `{client-provisioning-template-type} default encrypted` and then click the arrow to add it to your operating system entry.
```